solr/solrj/src/java/org/apache/solr/common/cloud/ClusterState.java

```diff
@@ -51,6 +50,10 @@
   private Set<String> liveNodes;
   private Set<String> hostAllowList;
 
+  // Copy of constants from ZkStateReader. Keep in sync if changed
+  public static final String NODE_NAME_PROP = "node_name";
```

Comment:
I think that a good amount of those props in ZKStateReader could live elsewhere and not be duplicated (since they aren't specific to ZK).

This includes the URL_SCHEME in the file above.

Reply:
I thought about the same. Some CloudConstants file that can be part of common, and be used even by ZkStateReader.